things/redis/streams_test.go

```diff
@@ -36,6 +36,7 @@ const (
 	channelCreate = channelPrefix + "create"
 	channelUpdate = channelPrefix + "update"
 	channelRemove = channelPrefix + "remove"
+	Prefix        = "fe6b4e92-cc98-425e-b0aa-"
```

Comment:
Is this used? Does it have to be exported?

Reply:
Correct.. I saw that it is not getting used.  just removed it.